conmon/conmon.c

```diff
@@ -618,7 +618,12 @@ static bool read_stdio(int fd, stdpipe_t pipe, bool *eof)
 
 static void on_sigchld(G_GNUC_UNUSED int signal)
 {
-	raise (SIGUSR1);
+	raise(SIGUSR1);
+}
+
+static void on_sig_exit(int signal)
+{
+	exit(128 + signal);
```

Comment:
Can we try and kill the container here, if it's successfully been made? We can live without an exit file easily enough, but killing the child PID to ensure we don't have unmonitored `runc` processes would be nice